libmscore/textedit.cpp

```diff
@@ -454,7 +455,13 @@ bool TextBase::edit(EditData& ed)
                               return true;
                         }
                   }
-            }
+            if (ctrlPressed && altPressed) {
+                  if (ed.key == Qt::Key_hyphen) {
+                        insertSym(ed, SymId::lyricsElision);
+                        return true;
+                        }
+                  break;
+                  }
```

Comment:
still a closing brace missing here

Reply:
is it the same I shouldn't have deleted?